### PR TITLE
sdk/js: add attestFromSui

### PR DESCRIPTION
**Pull request by heyitaki**
This PR adds attest functionality for the WH Sui SDK

### CODE REVIEW

**Comment by heyitaki** [comment]
Current dependencies on/for this PR:
* sui/integration_v2
  * **PR #2628** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/2628" target="_blank">Graphite</a>  👈
    * **PR #2632** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/2632" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/2628?utm_source=stack-comment).

File: sdk/js/src/token_bridge/attest.ts [hunk 1]
```diff
@@ -344,3 +349,34 @@ export function attestFromAptos(
...
   return attestTokenAptos(tokenBridgeAddress, tokenChain, tokenAddress);
 }
+
+export async function attestFromSui(
+  provider: JsonRpcProvider,
+  tokenBridgeAddress: string,
+  coreBridgeStateObjectId: string,
+  tokenBridgeStateObjectId: string,
+  coinType: string,
+  feeAmount: number
```
Comment (kev1n-peters) [inline sdk/js/src/token_bridge/attest.ts:359]: Should this be a `bigint`?

File: sdk/js/src/token_bridge/attest.ts [hunk 2]
```diff
@@ -344,3 +349,34 @@ export function attestFromAptos(
...
+  const tx = new TransactionBlock();
+  const [feeCoin] = tx.splitCoins(tx.gas, [tx.pure(feeAmount)]);
+  tx.moveCall({
+    target: `${tokenBridgeAddress}::attest_token::attest_token`,
+    arguments: [
+      tx.object(tokenBridgeStateObjectId),
+      tx.object(coreBridgeStateObjectId),
+      feeCoin,
+      tx.object(metadata.id),
+      tx.pure(69),
```
Comment (kev1n-peters) [inline sdk/js/src/token_bridge/attest.ts:376]: Does the nonce matter?

**Review by evan-gray — APPROVED** [review]
Good enough for integrations. Glad to have fixed up the test utils stuff.

**Pull request merged**